OloEngine/tests/MCP/McpClientStdioTest.cpp

```
@@ -0,0 +1,345 @@
+        {
+            if (OnLineSink)
+                OnLineSink(message.dump());
+        }
+
+        void Close() override
+        {
+            if (!m_Closed.exchange(true) && OnClosedSink)
+                OnClosedSink();
+        }
+
+        [[nodiscard]] bool WroteMethod(const std::string& method) const
```

Cppcheck (2.21.0)

[style] The function 'WroteMethod' is never used.

(unusedFunction)